Header header 

int32 num_linesegments
geometry_msgs/Point[] start 
geometry_msgs/Point[] end
